test/ui-test/src/screens/StatusWelcomeScreen.py

```diff
@@ -105,24 +106,31 @@ def input_username_and_password_and_finalize_sign_up(self, username: str, passwo
         self.input_confirmation_password(password)
 
         if sys.platform == "darwin":
-            click_obj_by_name(SignUpComponents.PASSWORD_PREFERENCE.value)
+            while(is_loaded_visible_and_enabled(SignUpComponents.PASSWORD_PREFERENCE.value, 10)[0]):
```

Comment:
Just wondering what would happen if this is always false? Does squish have some predefined timeout per test case or it will hang forever?

Reply:
Great catch. I will fix it using the approach here https://github.com/status-im/status-desktop/blob/397f5a1b1a5a3c2369cc1e0c359b8c7c10365e12/test/ui-test/src/drivers/SquishDriver.py#L171 after the PR is merged. Thanks